app/Models/Store/Product.php

```diff
@@ -169,6 +169,12 @@ public function release($quantity)
             return true;
         }
 
+        // stock may have been directly updated to 0.
+        // TODO: should count reservations and available stock separately or something.
+        if ($this->stock <= 0) {
+            return false;
```

Comment:
wtf does the return do ಠ_ಠ (also in the one above)

Reply:
*return value

Reply:
SUCCE-..wait the last one doesn't return 😕

Reply:
looks like the original function returned the save return and then nuked when changed to use increment.